public/stylesheets/site/2.0/25-media-midsize.css

```diff
@@ -0,0 +1,47 @@
+/* MEDIA: only screen and (max-width: 62em), handheld ENDMEDIA */
```

Comment:
The dashboard and main stuff is just moving the horizontal dashboard styling out of the narrow stylesheet and into this one -- the style itself is unchanged.